sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1399,52 +1399,6 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
-  test(
```

Comment:
@viirya 

In the future, it should not be difficult to look through the history to find the code.

Besides, the future implementation and expectation of `create temporary table using .. as query` probably will be different than it is now. So, maybe removing it for now creates less confusion?